### DISCUSSION TITLE
How can I configure keybinds for my G502 Hero in my i3 config file?

### DISCUSSION THREAD

**Discussion by rosidae** [i3/i3#5672] [Q&A] [answered]
i3 v4.22
Ubuntu 22.04.3 LTS

I have a G502 hero and I would like to set keybinds for the extra buttons on it.
More specifically I would like to make them do the following
* Pause & Play - DPI switcher
* Next/Previous - Scroll wheel left/right

I already know how i'll accomplish this but I dont know the names of these buttons nor do I know how to find them. (I'll use dbus-send to control spotify)
I looked at the website and all I found was example config files with no actual explaination on what buttons are what.
(volume control already works with onboard memory, i am dualbooting windows & ubuntu so I have access to the aforementioned onboard memory)

**Comment by orestisfl** [comment, marked as answer]
You can find the name of keys using `xev`
